modules/number-picker/main/index.coffee

```diff
@@ -62,8 +61,9 @@ class NumberPicker extends hx.EventEmitter
     @selectInput = selection.append('input')
     @selectInput.attr('type', 'number')
     @selectInput.on 'blur', 'hx.number-picker', =>
-      @emit 'input-change', {value: @value()}
-      @value(undefined, @selectInput.value())
+      if @selectInput.attr('readonly') is undefined
```

Comment:
The events were being fired when the input was readonly, causing issues with localised number values (e.g. time picker)